extensions/cornerstone/src/components/CinePlayer/CinePlayer.tsx

```diff
@@ -111,7 +110,10 @@ function WrappedCinePlayer({
     return () => {
       cineService.setCine({ id: viewportId, isPlaying: false });
 
-      eventTarget.removeEventListener(Enums.Events.STACK_VIEWPORT_NEW_STACK, newDisplaySetHandler);
+      enabledVPElement.removeEventListener(
+        Enums.Events.VIEWPORT_NEW_IMAGE_SET,
+        newDisplaySetHandler
+      );
       enabledVPElement.removeEventListener(
```

Comment:
Does it still need the new volume listener?  Seems to me it should have the new image set triggered for new volume as well.

Reply:
Seems like i forgot to migrate all viewports to this event, so cornerstone3D 3.0 candidate